### PR TITLE
Add .gitignore; includes Rubymine/IDEA config

### PR DESCRIPTION
**Pull request by jottinger**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,32 +1,13 @@
...
-
-# Ignore master key for decrypting credentials and more.
-/config/master.key
-
-/public/packs
-/public/packs-test
-/node_modules
-/yarn-error.log
-yarn-debug.log*
-.yarn-integrity
```
Comment (betacar) [inline .gitignore]: Is there a reason on why these were deleted?

Comment (jottinger) [reply .gitignore]: Um, mostly because apparently my copy was out of date when I committed it. They should DEFINITELY be added back in.

Comment (jottinger) [reply .gitignore]: Okay, I'm going to revise the branch entirely - give me a few minutes, I'm going to work with a better delta. I'm not sure where the original went on my filesystem.

File: .gitignore [hunk 2]
```diff
@@ -30,3 +30,9 @@ yarn-debug.log*
 db/.data
 coverage
 .DS_Store
+
+Gemfile.lock
```
Comment (betacar) [inline .gitignore]: We shouldn't ignore the lock file.

**Pull request closed without merging**